app/jobs/search_keyword_job.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class SearchKeywordJob < ApplicationJob
+  queue_as :default
+
+  def perform(keyword_id)
+    keyword = Keyword.find(keyword_id)
+    html = GoogleSearchService.new(keyword: keyword.keyword).call
+    utf_8_html = html.force_encoding('iso8859-1').encode('utf-8')
+    update_keyword(keyword, utf_8_html)
+  rescue ActiveRecord::RecordNotFound, GoogleSearch::Errors::SearchKeywordError, ActiveRecord::StatementInvalid
+    update_keyword_status keyword, :failed
+  end
+
+  private
+
+  def update_keyword_status(keyword, status)
+    keyword.update(status: status)
+  end
+
+  def update_keyword(keyword, html)
+    keyword.update({ html: html, status: :completed })
```

Comment:
Just a question, currently we are updating only `html` and `status` column, we will update `total_links_count`, `ads_top_count`, later PRs, right?

Reply:
Yes, we will parse data from HTML, save into total_links_count and another properties in another PR :D